package.json

```diff
@@ -63,7 +63,11 @@
     "travis-after-all": "1.4.4",
     "webpack": "2.2.1",
     "webpack-combine-loaders": "2.0.3",
+    "webpack-dashboard": "^0.3.0",
```

Comment:
I'll update these, thanks for mentioning. Wasn't sure how to include them. Also noticed this is unused, from trying to get it to work with webpack 2, but failing. Will remove for now.